etna/datasets/tsdataset.py

```diff
@@ -585,8 +585,8 @@ def plot(
         n_segments: int = 10,
         column: str = "target",
         segments: Optional[Sequence[str]] = None,
-        start: Optional[Union[pd.Timestamp, int, str]] = None,
-        end: Optional[Union[pd.Timestamp, int, str]] = None,
+        start: Union[pd.Timestamp, int, str, None] = None,
+        end: Union[pd.Timestamp, int, str, None] = None,
```

Comment:
Why do we change typing here?

Reply:
I think it is shorter to add `None` into `Union` than to wrap everything in `Optional`.